packages/engine/src/renderer/EngineRendererState.ts

```diff
@@ -59,6 +77,8 @@ export function EngineRendererReceptor(action: EngineRendererActionType) {
         return setUsePostProcessing(s, action.usePostProcessing)
       case 'WEBGL_RENDERER_SHADOWS':
         return setUseShadows(s, action.useShadows)
+      case 'WEBGL_SET_CONTROL_MODEL':
```

Comment:
this should be implemented in a new file, rather than here in the renderer settings. maybe in the /packages/engine/src/avatar/state/AvatarInputState.ts